### PR TITLE
feat: created easier-to-digest json representation

### PR DESCRIPTION
**Pull request by PeterStrob**
Signed-off-by: PeterStrob <peter@indicio.tech>

### CODE REVIEW

File: src/renderer/components/Verifications/Verification.vue [hunk 1]
```diff
@@ -15,15 +15,19 @@
     <el-collapse v-model="expanded_items">
       <ul class="list">
         <el-collapse-item
-          v-if="presentations.length"
```
Comment (burdettadam) [inline src/renderer/components/Verifications/Verification.vue:18]: Is this check really no longer needed?

Comment (burdettadam) [reply src/renderer/components/Verifications/Verification.vue:18]: I see, the `presentation_title` makes the check redundant.

**Review by burdettadam — APPROVED** [review]

File: src/renderer/components/ConnectionsNew/ConnectionList.vue [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <template >
   <div v-if="list.length">
-    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
```
Comment (dbluhm) [inline src/renderer/components/ConnectionsNew/ConnectionList.vue:3]: For now, I think we should refrain from changing themes for consistency's sake. If we decide to move to a dark theme, this should be done everywhere at once instead of piecemeal. For now, please revert this change.

File: src/renderer/components/Dids/index.vue [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <template >
   <el-row>
-  <nav class="navbar navbar-expand-lg navbar-light bg-light">
+  <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
```
Comment (dbluhm) [inline src/renderer/components/Dids/index.vue:3]: Same here

**Review by dbluhm — CHANGES_REQUESTED** [review]
Nice work, a couple of requested changes.

**Review by dbluhm — APPROVED** [review]
Nice work, thanks!
